docs/api/version-history.md

```diff
@@ -13,6 +13,12 @@ keywords: "API, Docker, rcli, REST, documentation"
      will be rejected.
 -->
 
+## v1.40 API changes
+
+[Docker Engine API v1.40](https://docs.docker.com/engine/api/v1.40/) documentation
+
+* `POST /containers/create` and `POST /services/create` now supports `BindOptions.NoRecursive`.
```

Comment:
Can you also mention;

- `GET /containers/<id>/json`
- `GET /containers/json`

And when swarmkit changes are done;

- `POST /services/create`
- `POST /services/{id}/update`
- `GET /services`
- `GET /services/{id}`